tests/test.validator.js

```diff
@@ -103,6 +103,7 @@ describe('Validator', function() {
   // - install.rdf
   it('should send JSScanner messages to the collector', () => {
     var addonValidator = new Validator({_: ['tests/example.xpi']});
+    addonValidator.print = sinon.stub();
```

Comment:
A comment to explain why we're doing this would be handy for the future. Normally I assume stubbing methods is for testing purposes, but this is really just a flourish so it'd be good to clarify why it's here.